ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.ts

```diff
@@ -2337,7 +2319,7 @@ const triggerSet: TriggerSet<Data> = {
 
         // Calculate which adjacent zone to go to, if needed
         let adjacentZone = null;
-        if (effectiveTemperature && adjacentZones) {
+        if (effectiveTemperature !== 0) {
```

Comment:
I'm not 100% sure on this change, it seems right at a glance but I've got no context for this fight since I've never done DR or DRS.

Reply:
Yeah, this is correct.  There's two cleaves, and the first time through (and sometimes the second?) you can resolve it by being hit by the first cleave (and then not needing the second).